### PR TITLE
Create Footer, Header and Hero Section (with placeholder)

### PR DESCRIPTION
**Pull request by Walsker**
Missing any use of the Button component, since @OmarG247's already working on that in #15.

### CODE REVIEW

**Comment by Walsker** [comment]
test

File: src/components/header.tsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
+import React, {useState} from 'react';
+import styled from 'styled-components';
+import {Link, NavLink, useRouteMatch} from 'react-router-dom';
```
Comment (dellisd) [inline src/components/header.tsx:3]:
```suggestion
import {Link, useRouteMatch} from 'react-router-dom';
```
Unused

**Comment by Walsker** [comment]
test 2

File: src/components/footer.tsx [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+        onClick={() =>
+          window ? window.scrollTo({top: 0, behavior: 'smooth'}) : {}
+        }
+      >
+        <FontAwesomeIcon icon={faArrowUp} size='sm' /> Back to Top
+      </BackToTopButton>
+      <Content>
+        <a
+          href='https://cuhacking.com'
+          rel='noopener referrer external'
```
Comment (dellisd) [inline src/components/footer.tsx:125]:
```suggestion
          rel='noopener noreferrer external'
```

**Review by dellisd — CHANGES_REQUESTED** [review]

**Review by dellisd — APPROVED** [review]

**Pull request merged**